moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -74,7 +74,7 @@ struct JumpThreshold
   double revolute;
```

Comment:
``factor``, above, should have better name or at least be documented in the comments above